crates/libs/registry/src/key.rs

```diff
@@ -37,6 +37,11 @@ impl Key {
         win32_error(result).map(|_| Self(handle))
     }
 
+    /// Returns the underlying registry key handle.
+    pub fn as_raw(&self) -> isize {
+        self.0
+    }
```

Comment:
This duplicates the key handle but `Key` also implements `Drop` potentially leaving callers with an invalid handle. Should this be `unsafe`?

Alternatively, this could be `pub fn into_inner(self) -> isize` instead making the transfer of ownership obvious.

Reply:
There is no transfer of ownership here. This is similar to https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.as_ptr where the underlying value is returned without changing ownership and requiring the caller to use it judiciously.